README.markdown

```diff
@@ -158,6 +155,21 @@ To use a mirror of the iojs binaries, set `$NVM_IOJS_ORG_MIRROR`:
 
 `nvm use` will not, by default, create a "current" symlink. Set `$NVM_SYMLINK_CURRENT` to "true" to enable this behavior, which is sometimes useful for IDEs.
 
+### .nvmrc
+
+You can create a `.nvmrc` file containing version number in the project root directory (or any parent directory).
+`nvm use`, `nvm install`, `nvm exec`, `nvm run`, and `nvm which` will all respect an `.nvmrc` file when a version is not supplied.
+
+For example, to make NVM default to version 5 for the current directory:
```

Comment:
Let's make this example `5.x`, and `5` in the nvmrc, so it's more useful (nobody should be using 5.0.0 :-) )